<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1350</width>
         <height>740</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>400</y>
          <width>50</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_add">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>400</y>
          <width>50</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>400</y>
          <width>50</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_admin">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>400</y>
          <width>50</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_csv">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>560</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>导出</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_looking">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>460</y>
          <width>50</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>460</y>
          <width>125</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>请输入查找的内容</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_looking">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>460</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>950</width>
          <height>300</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_95">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>340</y>
          <width>320</width>
          <height>360</height>
         </rect>
        </property>
        <property name="title">
         <string>负荷总功率</string>
        </property>
        <widget class="QGroupBox" name="groupBox_1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>300</width>
           <height>80</height>
          </rect>
         </property>
         <property name="title">
          <string>一类负荷总功率</string>
         </property>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>P:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Q:</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>300</width>
           <height>80</height>
          </rect>
         </property>
         <property name="title">
          <string>二类负荷总功率</string>
         </property>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>P:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Q:</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>260</y>
           <width>300</width>
           <height>80</height>
          </rect>
         </property>
         <property name="title">
          <string>三类负荷总功率</string>
         </property>
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>P:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Q:</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="pushButton_active_power">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>计算功率</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>1010</x>
          <y>180</y>
          <width>320</width>
          <height>520</height>
         </rect>
        </property>
        <property name="title">
         <string>负荷总功率（考虑同时系数）</string>
        </property>
        <widget class="QGroupBox" name="groupBox_97">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>430</y>
           <width>300</width>
           <height>70</height>
          </rect>
         </property>
         <property name="title">
          <string>考虑电网损失5%时所需功率</string>
         </property>
         <widget class="QLineEdit" name="lineEdit_22">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_25">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>P:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_23">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_26">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Q:</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_98">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>340</y>
           <width>300</width>
           <height>70</height>
          </rect>
         </property>
         <property name="title">
          <string>一、二类负荷功率总和（考虑同时系数）</string>
         </property>
         <widget class="QLineEdit" name="lineEdit_14">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>P:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_15">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Q:</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_99">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>250</y>
           <width>300</width>
           <height>70</height>
          </rect>
         </property>
         <property name="title">
          <string>二类负荷总功率（考虑同时系数）</string>
         </property>
         <widget class="QLineEdit" name="lineEdit_12">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>P:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_13">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Q:</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_100">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>300</width>
           <height>70</height>
          </rect>
         </property>
         <property name="title">
          <string>一类负荷总功率（考虑同时系数）</string>
         </property>
         <widget class="QLineEdit" name="lineEdit_10">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>P:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_11">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>30</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Q:</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>280</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>功率计算（考虑同时系数）</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_96">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>300</width>
           <height>90</height>
          </rect>
         </property>
         <property name="title">
          <string>同时系数</string>
         </property>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>125</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>二类负荷同时系数</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>120</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_9">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>60</y>
            <width>120</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>125</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>一类负荷同时系数</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>400</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>子窗口编辑</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>510</y>
          <width>300</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>工况选择</string>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>40</y>
           <width>70</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>运行工况</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>35</y>
           <width>90</width>
           <height>30</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>无选择</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>航行状态</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>起锚状态</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>停泊状态</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>装卸货状态</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>应急状态</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>600</y>
          <width>300</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="radioButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>30</y>
           <width>130</width>
           <height>20</height>
          </rect>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>负荷总功率计算</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>1020</x>
          <y>90</y>
          <width>300</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>240</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>负荷总功率计算（考虑同时系数）</string>
         </property>
        </widget>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>560</y>
          <width>200</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>520</y>
          <width>180</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>请先输入负荷表标题再导出</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
